action.yml

```diff
@@ -17,7 +17,7 @@ inputs:
     description: The number of the issue or pull request.
     required: false
 runs:
-  using: node12
+  using: node20
```

Comment:
I was going to look into making a similar change.

@aximov are you able to merge this change?